src/client/styles/components/_form-dropdown.scss

```diff
@@ -16,4 +16,14 @@
   &.has-error {
     border-color: $red;
   }
+
+  @include media(desktop){
+    padding: 9px 0px;
+  }
+  @include media(mobile){
```

Comment:
Don't need to enclose the mobile declarations within a mixin. Modules were build mobile first so change line 7 to the desired mobile value and use one mixin using `media(tablet) { ... }` to define tablet and desktop spacing.